dev-tools/composer.json

```diff
@@ -9,7 +9,7 @@
         "mi-schi/phpmd-extension": "^4.3.0",
         "phpcompatibility/php-compatibility": "10.x-dev as 9.99.0",
         "phpcompatibility/phpcompatibility-symfony": "^1.2.2",
-        "phpmd/phpmd": "^2.15.0",
+        "phpmd/phpmd": "dev-master as 2.16.0",
```

Comment:
:(

Reply:
Better this than having some violations that are not reported... until the file is modified.

After all, these are dev tools, plus we already are doing it for `phpcompatibility/php-compatibility`.